doc/api/cli.md

```diff
@@ -1662,7 +1663,6 @@ $ node --max-old-space-size=1536 index.js
 [debugger]: debugger.md
 [debugging security implications]: https://nodejs.org/en/docs/guides/debugging-getting-started/#security-implications
 [emit_warning]: process.md#process_process_emitwarning_warning_type_code_ctor
-[experimental ECMAScript Module loader]: esm.md#esm_experimental_loaders
```

Comment:
the `#esm_experimental_loaders` anchor should still be preserved, so links don't break.

Reply:
I've added these.